src/co2.js

```diff
@@ -5,19 +5,10 @@ import SustainableWebDesign from "./sustainable-web-design.js";
 
 class CO2 {
   constructor(options) {
-    this.options = options;
-
-    // default model
     this.model = new OneByte();
-
-    if (options) {
-      if (options.model === "swd") {
-        this.model = new SustainableWebDesign();
-      } else {
-        // retain the fallback for people developing
-        // new models that follow the same API
-        this.model = new options.model();
-      }
+    // Using optional chaining allows an empty object to be passed in without breaking the code.
+    if (options?.model === "swd") {
```

Comment:
Oh neat! I've never used optional chaining in js before, and this reads much better than what we had before 👍